src/components/Auth/Register.jsx

```diff
@@ -0,0 +1,245 @@
+import React, { useState } from "react";
+import Avatar from "@material-ui/core/Avatar";
+import Button from "@material-ui/core/Button";
+import CssBaseline from "@material-ui/core/CssBaseline";
+import TextField from "@material-ui/core/TextField";
+import Link from "@material-ui/core/Link";
+import Grid from "@material-ui/core/Grid";
+import LockOutlinedIcon from "@material-ui/icons/LockOutlined";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+import Container from "@material-ui/core/Container";
+
+// import { IconButton, Tooltip } from "@material-ui/core";
+// import { PhotoCamera } from "@material-ui/icons";
+
+import ErrorMessage from "../ErrorMessage";
+import Loading from "../Auth/Loading";
+import axios from "axios";
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    marginTop: theme.spacing(8),
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+  },
+  avatar: {
+    margin: theme.spacing(1),
+    backgroundColor: theme.palette.secondary.main,
+  },
+  form: {
+    width: "100%", // Fix IE 11 issue.
+    marginTop: theme.spacing(3),
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+  input: {
+    display: "none",
+  },
+  faceImage: {
+    color: theme.palette.primary.light,
+  },
+}));
+
+export default function Register() {
```

Comment:
```suggestion
const Register = () => {
  return ()
};
export default Register;
```
its better to write fonctional component
if you wanted to do the export in one line they explain here its not allowed in ES6 https://stackoverflow.com/questions/41695189/react-how-to-export-a-pure-stateless-component